docs/src/CNAME

```diff
@@ -1 +0,0 @@
-lsp.sublimetext.io
```

Comment:
This file is necessary.

See
https://github.com/sublimelsp/LSP/pull/1616#issue-836406285